source/applications/advanced/hand_eye_calibration/hand_eye_calibration.py

```diff
@@ -83,10 +106,9 @@ def _main():
         else:
             print(f"Unknown command '{command}'")
 
-    print("Performing Eye-to-Hand calibration")
-    calibration_result = zivid.calibration.calibrate_eye_to_hand(calibration_inputs)
+    calibration_result = _perform_calibration(hand_eye_input)
     code.interact(local=locals())
-    if calibration_result:
+    if calibration_result.valid():
         print("Eye-to-Hand calibration OK")
         print(f"Result:\n{calibration_result}")
```

Comment:
Change to general "Hand eye calibration failed"?

Reply:
Done in https://github.com/zivid/zivid-python-samples/pull/102/commits/46115ebef3083e25b98c4981a3e9e0238a10add2